events/2025-04-22-community-call-hub-cloud-costs/index.qmd

```diff
@@ -0,0 +1,53 @@
+---
+title: "Community Call: Beyond “it depends”: what we’re learning about cloud costs for Earth science workflows in our JupyterHub"
+description: something?
```

Comment:
```suggestion
description: Beyond "it depends" - learning about cloud costs for science by building reusable open tooling for monitoring and reporting
```

Reply:
This is one idea; a little redundant or long maybe?

Reply:
I like it a lot!